core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -493,8 +493,11 @@ func (fm *FluxMonitor) processLogs() {
 			err = broadcast.MarkConsumed()
 
 		case *flags_wrapper.FlagsFlagLowered:
-			fm.pollManager.Awaken(fm.initialRoundState())
-			fm.pollIfEligible(PollRequestTypeAwaken, NewZeroDeviationChecker())
+			// Only reactivate if it is hibernating
+			if fm.pollManager.cfg.IsHibernating {
```

Comment:
a bit minor, but you could move it to the inside of PollManager and then have:
```
if fm.pollManager.Awaken(fm.initialRoundState()) {
    fm.pollIfEligible(PollRequestTypeAwaken, NewZeroDeviationChecker())
}		
```
to make it a bit more clear, as FluxMonitor also has IsHibernating() method which is different